infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/devfile/KubernetesComponentToWorkspaceApplier.java

```diff
@@ -212,20 +212,21 @@ private void applyProjectsVolumes(List<PodData> podsData, List<HasMetadata> comp
     }
 
     for (PodData podData : podsData) {
+      final String podVolumeName = PROJECTS_VOLUME_NAME + "-" + podData.getMetadata().getName();
       if (podData
           .getSpec()
           .getVolumes()
           .stream()
           .noneMatch(volume -> volume.getName().equals(PROJECTS_VOLUME_NAME))) {
-        Volume volume = newVolume(PROJECTS_VOLUME_NAME, PROJECTS_VOLUME_NAME);
+        Volume volume = newVolume(podVolumeName, PROJECTS_VOLUME_NAME);
```

Comment:
I'm not clear why this is helpful, we can have the same volume mounted as multiple volumeMounts, so adding the same PVC as a volume with multiple names seems strange.